tests/actions/vstudio/vc2010/test_rule_vars.lua

```diff
@@ -100,11 +100,11 @@
 
 	function suite.onPathVar()
 		createVar { name="MyVar", kind="path" }
-		myRuleVars { MyVar = "../../path/to/file" }
+		myRuleVars { MyVar = "../path/to/file" }
```

Comment:
this is actually a separate bugfix that I hit.. if you go up to far, the path becomes absolute, and stays absolute. so ../.. actually goes up beyond the premake folder itself, and hence in our case where it is in "c:\dev\premake", the patch would actually become "C:\path\to\file", which then subsequently stays absolute.